l10nfr.properties

```diff
@@ -1,8 +1,8 @@
 category=Localization
 description=Language Pack for French
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-fr
-publicVersions=1.15.1,9.8.1,9.9.2,10.0.0
-archivedVersions=9.8.0,9.9.0,9.9.1
+publicVersions=9.9.2,10.0.0
+archivedVersions=9.8.0,9.9.0,9.9.1,1.15.1,9.8.1,
```

Comment:
Is this going to work with the trailing comma at the end?
Can you order the versions of archivedVersions?